wcfsetup/install/files/acp/update_com.woltlab.wcf_5.3_packageServer.php

```diff
@@ -0,0 +1,56 @@
+<?php
+namespace wcf\acp;
+use wcf\data\package\update\server\PackageUpdateServerEditor;
+use wcf\system\WCF;
+
+// Do not use the `PackageUpdateServer` classes because we need to access
+// the raw server URL that is implicitly rewritten in 5.3.
+$sql = "SELECT  *
+	FROM    wcf1_package_update_server
+	WHERE	LOWER(serverURL) REGEXP 'https?://(store|update)\.woltlab\.com/.*'";
+$statement = WCF::getDB()->prepareStatement($sql);
+$statement->execute();
+
+// Try to extract authentication credentials from the previous package servers.
+$newServers = [
+	"update" => [
+		"loginUsername" => "",
+		"loginPassword" => "",
+	],
+	"store" => [
+		"loginUsername" => "",
+		"loginPassword" => "",
+	],
+];
+
+$deleteServerIDs = [];
+while ($row = $statement->fetchArray()) {
+	$deleteServerIDs[] = $row["packageUpdateServerID"];
+	
+	// Only use values from the "2019" servers to avoid dealing with outdated
+	// credentials that have never been updated.
+	$serverURL = $row["serverURL"];
+	if (preg_match("~^https?://update\.woltlab\.com/2019/~", $serverURL)) {
```

Comment:
Shouldn't you preserve the 5.3 update servers? They must have been added for the upgrade anyway.

Thinking about this: It should be sufficient to simply add something like `AND serverURL NOT LIKE '%5.3/'` to the query in the SQL file to fix the issue, no?

Reply:
Yes, you are correct, that is much easier.